lib/ethpy/ethpy/base/transactions.py

```diff
@@ -333,10 +338,8 @@ async def async_wait_for_transaction_receipt(
                 if tx_receipt is not None:
                     break
                 await _timeout.async_sleep(poll_latency)
-                # Exp backoff
-                poll_latency *= backoff
-                # Add random latency to avoid collisions
-                poll_latency += random.uniform(0, 1)
```

Comment:
lets put this one back in -- this can help avoid collisions. 0, 1 second is a big range though, so for speedup lets do `0, 0.1`

Reply:
done in 916c248a7d9a9f3f45aef2079e75da332b01a5f1